event-manager/src/main/java/evrentan/community/eventmanager/entity/EventVenue.java

```diff
@@ -0,0 +1,54 @@
+package evrentan.community.eventmanager.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.util.UUID;
+
+/**
+ * Event Venue Class represents the event_community table in the database.
+ * It is represented by {@link evrentan.community.eventmanager.dto.EventVenueDto} in DTO level.
+ *
+ * @author <a href="https://github.com/Onuraktasj">Onur Aktas</a>
+ * @since 1.0.0
+ */
+@Entity
+@Table(name = "event_venue")
+@Getter
+@Setter
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@NoArgsConstructor
+@EqualsAndHashCode
+@ToString
+@Builder
+public class EventVenue {
+
+    /**
+     * Community ID in UUID Type.
+     * It is equalized to the id column in the database.
+     * It is the primary key of the table.
+     * It is auto-generated by the database.
+     * It is unique.
+     * It is not nullable.
+     *
+     * @since 1.0.0
+     */
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id")
+    private UUID id;
+
+    /**
+     * Indicates whether the event venue is active or not.
+     * It is equalized to the is_active column in the database.
+     *
+     * @since 1.0.0
+     */
+    @Basic
+    @Column(name = "venue_id")
+    private boolean isActive;
+
+    @OneToOne
+    private EventEntity evenId;
```

Comment:
I do not recommend relations at the entity level. Just put in the foreign key and take the required entity when needed. This should be applied all over the full service implementation.